tools/multiqc.wdl

```diff
@@ -39,8 +39,11 @@ task multiqc {
         echo "~{sep('\n', input_files)}" > file_list.txt
 
         # Start YAML generation
-        echo "extra_fn_clean_exts:" > multiqc_config.yaml
-        
+        touch multiqc_config.yaml
+        if [ "~{(length(extra_fn_clean_exts) > 0)}" = "true" ] || [ "~{(length(mosdepth_labels) > 0)}" = "true" ]; then
```

Comment:
Nice catch on this. But bad timing on your fix, because I'm opening a PR which removes _all_ this logic haha.